filebeat/docs/autodiscover-docker-config.asciidoc

```diff
@@ -0,0 +1,39 @@
+Filebeat supports templates for both prospectors and modules.
+
+["source","yaml",subs="attributes"]
+-------------------------------------------------------------------------------------
+filebeat.autodiscover:
+  providers:
+    - type: docker
+      templates:
+        - condition:
+            equals:
+              docker.container.image: redis
+          config:
+            - type: docker
+              container.ids:
+                - "${data.docker.container.id}"
+              exclude_lines: ["^\\s+[\\-`('.|_]"]  # drop asciiart lines
+-------------------------------------------------------------------------------------
+
+This will launch `docker` logs prospector for all containers running a image with `redis` in its name.
+
+When defining modules, you can override default prospector to use docker:
```

Comment:
Change to:

If you are using modules, you can override the default prospector and use the `docker` prospector instead.